cli_tools_tests/e2e/gce_windows_upgrade/test_suites/windows_upgrade/windows_upgrade_test_suite.go

```diff
@@ -87,6 +88,20 @@ func TestSuite(
 		testsMap[testType][testBYOL] = runWindowsUpgradeBYOL
 	}
 
+	// These tests only apply to wrapper since it's testing staging install media.
+	staging2012 := junitxml.NewTestCase(
+		testSuiteName, fmt.Sprintf("[%v] %v", e2e.Wrapper, "Staging test 2008 -> 2012"))
+	staging2016 := junitxml.NewTestCase(
+		testSuiteName, fmt.Sprintf("[%v] %v", e2e.Wrapper, "Staging test 2012 -> 2016"))
+	staging2019 := junitxml.NewTestCase(
+		testSuiteName, fmt.Sprintf("[%v] %v", e2e.Wrapper, "Staging test 2012 -> 2019"))
+	staging2022 := junitxml.NewTestCase(
+		testSuiteName, fmt.Sprintf("[%v] %v", e2e.Wrapper, "Staging test 2012 -> 2022"))
```

Comment:
If the second element in the string format is fixed then you can move it to the format itself.